site/content/docs/monitoring/false-positives.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Understanding and managing false positives
+weight: 10
+menu:
+  platform:
+    parent: "Monitoring"
+---
+
+At Checkly, we provide a reliable and performant solution for running synthetic monitoring and API checks across 22 regions. Our infrastructure is designed to be robust, but sometimes, unforeseen issues like server or network problems can crop up—whether on our side or yours. Often, these issues show up as isolated incidents that resolve on their own. However, in the worst cases, they can cause false positives in your check results, where Checkly might report that your site or server is slow or unavailable, even though you know it's up and running.
+
+In this section, we’ll take a closer look at why these intermittent issues can occur and how you can protect your monitoring setup from them.
+
+## Common Causes of Performance Variations
+
+Networking is complex, and hiccups happen. These can occur anywhere—within our infrastructure, on your end, or somewhere in between—and don’t necessarily mean that your server or site is actually down. Here are some common factors that can affect performance:
```

Comment:
same here. I'd rewrite to more general terms "Networking is complex, the following things must be considered when setting up your monitoring strategy" something like that.